hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -201,8 +207,8 @@ setupNodeAndTUI' lovelace action =
                         nodeSocket
                     , cardanoNetworkId =
                         networkId
-                    , cardanoSigningKey =
-                        (cardanoSigningKey :: ChainConfig -> FilePath) chainConfig
+                      -- NOTE: TUI Client uses this key to do the external commit
```

Comment:
Should: comment in haddock when possible

Documenting things in haddock is better discoverable, and this comment in particular is relevant for other usage sites as well.